backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -695,6 +704,53 @@ public FutureExperimentRunDAO(
                                                     hyperparams.get(builder.getId()))),
                                     executor);
 
+                            final var futureHyperparamsFromConfigBlobs =
```

Comment:
Please comment this block of code. It's not trivial on what it's doing.

Reply:
Added comment on code for this block